NuRadioReco/detector/detector_browser/open_file.py

```diff
@@ -342,7 +343,7 @@ def toggle_open_button_active(filename, detector_type, default_station, need_def
 
 @app.callback(
     Output('detector-time-div', 'style'),
-    [Input('load-detector-button', 'n_clicks'),
+    [Input('output-dummy', 'children'),
```

Comment:
dont understand this change but okay.

Reply:
The previous trigger led to a race condition - it triggered on the same trigger that loads the detector, so if it was quicker than the other callback, detector_provider.get_detector() would still return None. Now it triggers on the output of the callback that loads the detector.